### PR TITLE
Huaxi/fluent folder top

### PR DESCRIPTION
**Pull request by yumikohey**
#### Pull request checklist

- [X ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

#### Description of changes
Added a css class to enableFluentIcon feature in content's padding.

#### Focus areas to test
Fluent Folder Cover Icon
(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/7998)

### CODE REVIEW

File: packages/experiments/src/components/FolderCover/FolderCover.scss [hunk 1]
```diff
@@ -53,6 +53,10 @@
   .hideContent & {
     opacity: 0;
   }
+
+  .isFluent & {
```
Comment (ThomasMichon) [inline packages/experiments/src/components/FolderCover/FolderCover.scss]: You can now change the `&.isFluent` rule in `.metadata` to `.isFluent & {` and then stop passing `isFluent` to the `.metadata` element directly.

File: packages/experiments/src/components/FolderCover/FolderCover.tsx [hunk 2]
```diff
@@ -95,7 +95,8 @@ export class FolderCover extends React.Component<IFolderCoverProps, IFolderCover
           [`ms-FolderCover--isLarge ${FolderCoverStyles.isLarge}`]: size === 'large',
           [`ms-FolderCover--isDefault ${FolderCoverStyles.isDefault}`]: type === 'default',
           [`ms-FolderCover--isMedia ${FolderCoverStyles.isMedia}`]: type === 'media',
-          [`ms-FolderCover--hideContent ${FolderCoverStyles.hideContent}`]: hideContent
+          [`ms-FolderCover--hideContent ${FolderCoverStyles.hideContent}`]: hideContent,
+          [`ms-FolderCover--isFluent ${FolderCoverStyles.isFluent}`]: isFluent
```
Comment (ThomasMichon) [inline packages/experiments/src/components/FolderCover/FolderCover.tsx]: You also need to modify `getFolderCoverLayout` to check `folderCoverProps.isFluent` and, if so, return different sizes.

File: packages/experiments/src/components/FolderCover/FolderCover.tsx [hunk 3]
```diff
@@ -125,13 +124,19 @@ export interface IFolderCoverLayout {
   contentSize: ISize;
 }
 
-export function getFolderCoverLayout(element: JSX.Element): IFolderCoverLayout {
+export function getFolderCoverLayout(element: JSX.Element, isFluent?: boolean): IFolderCoverLayout {
```
Comment (kenotron) [inline packages/experiments/src/components/FolderCover/FolderCover.tsx]: It seems really odd to me to see something like this as a flag. I would have expected something like this to come from a theme token or something. @JasonGore

Comment (bigbadcapers) [reply packages/experiments/src/components/FolderCover/FolderCover.tsx]: this is something we've added to support the new "Fluent FileType Icons", @ThomasMichon and Edgar (dunno his git handle) are in the loop

Comment (JasonGore) [reply packages/experiments/src/components/FolderCover/FolderCover.tsx]: I didn't know about `isFluent` classnaming in general, but to echo Ken's concern, adding component arguments and logic for particular themes won't scale. I'm really concerned about adding the `isFluent` argument to component code.

I have a few questions:
1. Where would the `isFluent` arg come from in code? If at all possible this should be isolated to styling / parent classNaming / CSS.
1. Are there any plans to convert FolderCover to use JS styling?

Not sure if @Jahnp would have any input either.

**Comment by yumikohey** [comment]
> I'm blocking here until someone else more familiar with our theming strategy also looks at this.

@kenotron  I talked to @JasonGore . He agrees that we can merge in this PR for now. 

Upcoming, I will submit a PR to have Fluent theme for CoverFolder; and remove `isFluent` props in `CoverFolder` component.

cc @ThomasMichon

**Pull request merged**